### PR TITLE
Track source based stream tracker config

### PR DESCRIPTION
**Pull request by boks1971**

### CODE REVIEW

File: pkg/sfu/streamtrackermanager.go [hunk 1]
```diff
@@ -33,16 +75,22 @@ func (s *StreamTrackerManager) OnAvailableLayersChanged(f func(availableLayers [
...
-	cyclesRequired := uint64(60) // 30s of continuous stream
-	if layer == 0 {
-		// be very forgiving for base layer to account for cases like static screen share where there could be only one packet per second
-		samplesRequired = 1
-		cyclesRequired = 1 // 1 packet in 2 seconds
-		cycleDuration = 2 * time.Second
-	}
-	tracker := NewStreamTracker(s.logger, samplesRequired, cyclesRequired, cycleDuration)
+	var params StreamTrackerParams
+	if s.source == livekit.TrackSource_SCREEN_SHARE || s.source == livekit.TrackSource_SCREEN_SHARE_AUDIO {
```
Comment (davidzhao) [inline pkg/sfu/streamtrackermanager.go:79]: screen share audio wouldn't go through here? since only video is subject to stream tracker

**Review by davidzhao — APPROVED** [review]
lg!

File: pkg/sfu/streamtrackermanager.go (hunk 1 above)
Comment (boks1971) [reply pkg/sfu/streamtrackermanager.go:79]: Oh, is that the audio track captured with screen share? I thought it was a special attribution for screen share when captured with audio included. Then, I need to remove that check in a couple of places. Let me do that.

Comment (boks1971) [reply pkg/sfu/streamtrackermanager.go]: Made relevant changes in this commit - https://github.com/livekit/livekit-server/pull/503/commits/1df74622537a6be80eb72201398c344dfe1fad35

Comment (davidzhao) [reply pkg/sfu/streamtrackermanager.go]: yup, that's it! 👍

**Pull request merged**